From 5cc94c851c06895bbd63fc081ced9fa0ca87e97d Mon Sep 17 00:00:00 2001
From: Peng Fan <Peng.Fan@freescale.com>
Date: Fri, 29 May 2015 13:59:02 +0800
Subject: [PATCH] MLK-10979-2 imx: dts: mx7d add ocotp fuse node

Add ocotp fuse node, and change clk from dummy to IMX7D_OCOTP_CLK

Signed-off-by: Peng Fan <Peng.Fan@freescale.com>
---
 arch/arm/boot/dts/imx7d.dtsi | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 47cda7d..959c0bc 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -578,7 +578,13 @@
 			ocotp: ocotp-ctrl@30350000 {
 				compatible = "syscon";
 				reg = <0x30350000 0x10000>;
-				clocks = <&clks IMX7D_CLK_DUMMY>;
+				clocks = <&clks IMX7D_OCOTP_CLK>;
+			};
+
+			ocotp-fuse@30350000 {
+				compatible = "fsl,imx7d-ocotp";
+				reg = <0x30350000 0x10000>;
+				clocks = <&clks IMX7D_OCOTP_CLK>;
 			};
 
 			anatop: anatop@30360000 {
-- 
1.8.0

